src/components/RulesModal.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div>
+      <div class="modal-dialog modal-lg" role="document">
+        <div class="modal-content">
+          <div class="modal-header">  <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+            <h3 class="modal-title">Programming Wars Rules</h3>
+          </div>
+          <div class="modal-body" style="text-align: left">
+            <h4>Introduction</h4>
+            <p>The basic elements of a program are simple to understand, but can be used to create complex
+              computer programs. In this fun, fast-paced game you try to complete a simple program before
+              the opponent player or team.</p>
+
+            <h4>Components</h4>
+            <p>There are five types of cards:</p>
+            <ul>
+              <li><h5>Instruction</h5>
+              <p>Instruction cards provide the instructions used to create the
+                program. There are 24 Instruction cards divided into three types: Instruction-1,
+                Instruction-2, and Instruction-3. There are 8 cards of each Instruction type.</p></li>
+              <li><h5>Repetition</h5>
+                <p>Repetition cards allow you to repeat a set of instructions. There are 13
+                  Repetition cards divided into 5 types:Repetition-2, Repetition-3, Repetition-4,
+                  and Repetition-5. There are 5 cards of the variable Repetition-X card.</p></li>
+              <li><h5>Group</h5>
+                <p>A Group card lets you combine two Instruction cards into one card. Group cards
+                  give you more points at the end of the game. </p></li>
+              <li><h5>Variables</h5>
+                <p>Variable cards are used with the variable Repetition (R x ) cards. You use
+                  these cards to change the number of times an Instruction or Group card repeats. There
+                  16 Variable cards divided into 5 types: Variable-2 (4 cards), Variable-3 (4 cards),
+                  Variable-4 (4 cards), Variable-5 (2 card), and Variable-6 (2 cards).</p></li>
+              <li><h5>Hack</h5>
+                <p>Hacking cards allow you to remove one of an opponent’s Instruction cards
+                  and put it in the discard pile. There are 4 Hacking cards.</p></li>
+            </ul>
+
+            <h4>Game Play</h4>
```

Comment:
I feel the "Card Playing Rules" section could be removed as most of the details are already in the individual card description. Seems a bit redundant. This would make the popup shorter.